src/components/labels/Label.stories.jsx

```diff
@@ -1,16 +1,46 @@
 import * as React from 'react';
-import Label, {ICON_TYPE, LABEL_COLORS_SET, LABEL_TYPE} from './Label';
-import Flex from '../flex/Flex';
+import Label, {
+  ICON_TYPE,
+  LABEL_COLORS_SET,
+  LABEL_TYPE,
+  COLORS_DEFAULT_MAP,
+  COLORS_SOLID_MAP,
+} from './Label';
+import {StoryVariantTable} from '../../_docs/utils';
+import Headline from '../text/Headline';
+import hex from '../colors/hex';
+
+const onCloseMock = () => null;
```

Comment:
nitpick: maybe just:

`function noop() {}` instead of those 2 mock functions?

Reply:
Good idea! 👍